### ISSUE TITLE
Segfault when using @ri.rternalize

### ISSUE THREAD

**Issue by DE0CH** [rpy2/rpy2#971]
**Describe the issue or bug**

When I make a function using `rpy2.rinterface.rternalize` without importing any other module, it causes a segfault.

**To Reproduce**

```
import rpy2.rinterface as ri 

@ri.rternalize
def f(x):
    return x
```

**Expected behavior**
I expect it to not segfault. Even if somehow something is not imported or not initialized, it should throw a python exeception and not cause a hard crash with a segfault.

**Error**
```
[1]    8704 segmentation fault  python3 test.py
```

**Additional context**
```
rpy2 version:
3.5.6
Python version:
3.9.2 (default, Feb 28 2021, 17:03:44) 
[GCC 10.2.1 20210110]
Looking for R's HOME:
    Environment variable R_HOME: None
    Calling `R RHOME`: /usr/lib/R
    Environment variable R_LIBS_USER: None
R's additions to LD_LIBRARY_PATH:
/usr/lib/R/lib:/usr/lib/x86_64-linux-gnu:/usr/lib/jvm/default-java/lib/server
R version:
    In the PATH: R version 4.0.4 (2021-02-15) -- "Lost Library Book"
    Loading R library from rpy2: OK
Additional directories to load R packages from:
None
C extension compilation:
  include:
  ['/usr/share/R/include']
  libraries:
  ['R', 'pcre2-8', 'lzma', 'bz2', 'z', 'rt', 'dl', 'm', 'icuuc', 'icui18n']
  library_dirs:
  ['/usr/lib/R/lib']
  extra_compile_args:
  []
  extra_link_args:
  ['-Wl,--export-dynamic', '-fopenmp', '-Wl,-z,relro']
```

**Comment by lgautier** [comment]
I can reproduce.

Yes, ideally if should fail with an error rather than segfault like it does for example with
```python
import rpy2.rinterface as ri
ri.globalenv['x']
```

**Comment by lgautier** [comment]
Fixed. It will be included in release 3.5.7 (which should happen over this weekend).
